src/Symfony/Bundle/FrameworkBundle/Tests/Routing/ServiceRouterLoaderTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Routing;
+
+use Symfony\Bundle\FrameworkBundle\Routing\ServiceRouterLoader;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+class ServiceRouterLoaderTest extends \PHPUnit_Framework_TestCase
+{
+    public function testLoadCallsServiceAndReturnsCollection()
+    {
+        $routeLoader = $this->getMock('Symfony\Bundle\FrameworkBundle\Routing\RouteLoaderInterface');
+
+        $container = $this->getMock('Symfony\\Component\\DependencyInjection\\Container', array('get'));
+
+        $container
+            ->expects($this->once())
```

Comment:
I wouldn't use `$this->once()` in case of stubs, `$this->any()` should be sufficient. The same applies to the other stub in this test.

We shouldn't care how many times this method is called. By relying on number of calls we make assumptions about implementation. All is important here that if a method is called, it will return a stubbed value. Assertions at the end perform verifications.